backend/presentation/models.py

```diff
@@ -36,4 +37,19 @@ def __str__(self):
         return self.title
 
     class Meta:
-        ordering = ['-created_at']
+        ordering = ["-created_at"]
+
+
+class Notification(models.Model):
+    objects = NotificationManager()
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    message = models.CharField(max_length=200)
+    user = models.ForeignKey(
+        User,
+        related_name="notifications",
+        on_delete=models.CASCADE,
+    )
+    reviewed = models.BooleanField(default=False)
```

Comment:
```suggestion
class Notification(models.Model):
    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
    message = models.CharField(max_length=200)
    user = models.ForeignKey(
        User,
        related_name="notifications",
        on_delete=models.CASCADE,
    )
    reviewed = models.BooleanField(default=False)
    
    objects = NotificationManager()
```
Specify `objects` after all fields and separate them by a blank line.